packages/massa-proto-cli/src/index.ts

```diff
@@ -68,9 +88,21 @@ async function run() {
     out,
     publicApi,
   );
-
+  const bignumTypes = `- type:
```

Comment:
this is a workaround
add a FIXME and a console.warn() explaining that the yml in inlined because finding the file is currently failing